pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -20,66 +20,72 @@
     'gps' for Radio Occultation profiles
 tag
     Select profile type, or scintillation, one of:
-    {'ionprf', 'sonprf', 'wetprf', 'atmprf', 'scnlv1'}
-sat_id
+    ['ionprf', 'wetprf', 'atmprf', 'eraprf', 'gfsprf',
+    'ionphs', 'podtec', 'scnlv1']
+inst_id
     None supported
 altitude_bin
     Number of kilometers to bin altitude profiles by when loading.
     Currently only supported for tag='ionprf'.
 
-Note
-----
-- 'ionprf: 'ionPrf' ionosphere profiles
-- 'sonprf': 'sonPrf' files
-- 'wetprf': 'wetPrf' files
-- 'atmprf': 'atmPrf' files
-- 'scnlv1': 'scnLv1' files
-
 Warnings
 --------
 - Routine was not produced by COSMIC team
-- More recent versions of netCDF4 and numpy limit the casting of some variable
-  types into others. This issue could prevent data loading for some variables
-  such as 'MSL_Altitude' in the 'sonprf' and 'wetprf' files. The default
-  UserWarning when this occurs is
-  ::
-
-    'UserWarning: WARNING: missing_value not used since it cannot be safely
-    cast to variable data type'
 
 """
 
 import datetime as dt
-import logging
 import os
 import requests
-import shutil
-import sys
 import tarfile
 
 import numpy as np
 import netCDF4
 import pandas as pds
 import pysat
+from pysat import logger
 from pysat.utils import files as futils
 
-logger = logging.getLogger(__name__)
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
 
 platform = 'cosmic'
 name = 'gps'
-tags = {'ionprf': '',
-        'sonprf': '',
-        'wetprf': '',
-        'atmprf': '',
-        'scnlv1': ''}
-sat_ids = {'': ['ionprf', 'sonprf', 'wetprf', 'atmprf', 'scnlv1']}
+
+l1_tags = ['ionPhs', 'podTec', 'scnLv1']
+lower_l1_tags = [tag.lower() for tag in l1_tags]
+
+l2_tags = ['ionPrf', 'wetPrf', 'atmPrf', 'eraPrf', 'gfsPrf']
+lower_l2_tags = [tag.lower() for tag in l2_tags]
+
+tags = {'ionprf': 'Ionospheric Profiles',
+        'wetprf': 'Atmospheric profiles with moisture',
+        'atmprf': 'Atmospheric profiles without moisture',
+        'eraprf': 'ERA-40 Interim reanalysis data',
+        'gfsprf': 'NCEP operational analysis data',
+        'ionphs': 'Ionospheric excess phase',
+        'podtec': 'Absolute Total Electron Content and auxiliary data',
+        'sclnv1': 'S4 scintillation index and auxiliary data'}
+
+inst_ids = {'': lower_l2_tags,
```

Comment:
It would be clearer to have this be 'level_2', if level is going to be used in `inst_id`.  Not sure why it's included at all, though.

Reply:
level_1b data is in a different format and requires different processing. A tag check is a way. I can add the others to a level_2 group but my thinking was that the final geophysical data is the default.

Reply:
I think a tag check would be in keeping with what we do in other instruments.